doc/gov/chapters/chapter04.md

```diff
@@ -45,51 +45,51 @@ Any Workstream Leads or Co-Leads positions can be held by any Participant.
 ### 4.2.3 Nomination
 
 - Any individual can nominate themselves or another participant for a given lead/co-lead position (the nominee must accept nomination if was nominate by someone else) as long as they agree to CNTT ToR.
-- An individual can self-nominate themselves or another participant to multiple positions.
+- An individual can nominate themselves or another participant to multiple positions.
```

Comment:
would it better to say "one or more" instead of "multiple"

Reply:
done